opensrp-path-zeir/src/main/java/org/smartregister/path/util/FormUtils.java

```diff
@@ -69,7 +69,7 @@ public static String obtainUpdatedForm(JSONObject form, CommonPersonObjectClient
     }
 
     private static JSONArray getSpinnerKeys(String locations) {
-        return getSpinnerKeys(locations.split(","));
+        return getSpinnerKeys(locations!=null? locations.split(","): null);
```

Comment:
Return an empty array instead or add a null check before using **locations** in spinner